variables.tf

```diff
@@ -183,16 +185,26 @@ variable "installer_log_level" {
     default = "info"
 }
 
+variable "helpernode_repo" {
```

Comment:
Short descriptions for all the new variables you added in quickstart guide will help.
Thanks!

Reply:
Done